components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIManagerConfiguration.java

```diff
@@ -626,6 +523,135 @@ private void readChildElements(OMElement serverConfig,
         }
     }
 
+    /**
+     * Set property values for each gateway environments defined in the api-manager.xml config file
+     *
+     * @param environmentElem OMElement of a single environment in the gateway environments list
+     */
+    void setEnvironmentConfig(OMElement environmentElem) throws APIManagementException {
+
+        Environment environment = new Environment();
+        environment.setType(environmentElem.getAttributeValue(new QName("type")));
+        String showInConsole = environmentElem.getAttributeValue(new QName("api-console"));
+        if (showInConsole != null) {
+            environment.setShowInConsole(Boolean.parseBoolean(showInConsole));
+        } else {
+            environment.setShowInConsole(true);
+        }
+        String isDefault = environmentElem.getAttributeValue(new QName("isDefault"));
+        if (isDefault != null) {
+            environment.setDefault(Boolean.parseBoolean(isDefault));
```

Comment:
Can't we add more info to Default variable like set default what ? is it make sense ?